reporting/reporting/Purchase.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Purchase]
+(
+	[LocationID] INT NOT NULL
+    CONSTRAINT [FK_Purchase_Location] FOREIGN KEY ([LocationID]) REFERENCES [Location]([LocationID]),
```

Comment:
@Sreenitha-Sampath 
1. Keep all constraints grouped at one place
2. Constraint name should have column name like FK_Purchase_Location_LocationId